syncer/ddl.go

```diff
@@ -312,5 +312,7 @@ func (s *Syncer) clearOnlineDDL(tctx *tcontext.Context, targetTable *filter.Tabl
 type shardingDDLInfo struct {
 	name   string
 	tables [][]*filter.Table
-	stmt   ast.StmtNode
+	// sourceTables []*filter.Table
```

Comment:
why comment this?

Reply:
This will not commit this pr. I will remove it next commit.

BTW, I think it has two improvements:
1. `tables [][]*filter.Table` can split to `sourceTables []*filter.Table` and `targetTables []*filter.Table`;
2. In some cases, `tables [][]*filter.Table` can change to `tables []*filter.Table`, and you can see why in https://github.com/pingcap/dm/issues/2129

We can discuss it in https://github.com/pingcap/dm/issues/2129

Reply:
SGTM